server/services/prismic.js

```diff
@@ -472,8 +475,8 @@ export async function getExhibitionAndEventPromos(query, collectionOpeningTimes,
   });
 
   const exhibitionPromos = createExhibitionPromos(allExhibitionsAndEvents.results.filter(e => e.type === 'exhibitions'));
-  const permanentExhibitionPromos = exhibitionPromos.filter(e => !e.end);
-  const temporaryExhibitionPromos = filterPromosByDate(exhibitionPromos.filter(e => e.end), fromDate, toDate);
+  const permanentExhibitionPromos = filterCurrentExhibitions(exhibitionPromos.filter(e => e.format === 'permanent'), todaysDate);
```

Comment:
[re:](https://github.com/wellcometrust/wellcomecollection.org/pull/2635/files#r188743648) We need to try not to use `slug` -  we could always do a check on ID or title here, as it's business logic